pages/tasks/[dynamicId]/index.js

```diff
@@ -29,6 +30,9 @@ export default function TaskDetailsPage() {
         deadline={task.deadline}
         priority={task.priority}
       />
+      <Link href={`/tasks/${dynamicId}/edit`} passHref legacyBehavior>
+        <StyledButton>edit</StyledButton>
+      </Link>
```

Comment:
To combine links and buttons is something we do not want as links and buttons each have different use cases. [See this article to learn more about when to use a link or a button.](https://css-tricks.com/a-complete-guide-to-links-and-buttons/). Instead just style the link and let it look like a button. 💪

Reply:
That's what I thought. Will fix it too )))